src/Swarm/Language/Syntax.hs

```diff
@@ -323,12 +323,15 @@ data Const
     AppF
   | -- God-like sensing operations
 
-    -- | Run a command as if you were another robot.
+    -- | Hypothetically run a command as if you were another robot.
+    --   Make sure this is the LAST command!
```

Comment:
What does "Make sure this is the LAST command" mean?

Reply:
Deleted. All commands are now commands, as they should be. :sweat_smile: